TowerForge/TowerForge/Networking/GamePlayer.swift

```diff
@@ -8,10 +8,10 @@
 import Foundation
 
 class GamePlayer: Codable {
-    var userPlayerId: UserPlayerId?
+    var userPlayerId: UserPlayerId
     let userName: String
 
-    init(userPlayerId: UserPlayerId? = nil, userName: String) {
+    init(userPlayerId: UserPlayerId = UUID().uuidString, userName: String) {
```

Comment:
Isn't this `UserPlayerId` meant to be from the Firebase generated userId?

Reply:
Yep. Originally this was the case but sticking to that will make it hard for me to be able to assign the player as a host if there isn't one assigned yet. So i converted it to this. It shouldn't affect the behaviour for everything else.